### ISSUE TITLE
References missing when merging

### ISSUE THREAD

**Issue by JulianGR** [pwndoc-ng/pwndoc-ng#46]
TL;DR: When merging 2 vulnerabilities from different languages, in one of the languages, the references disappear:

1.- The vulnerability in spanish:
![image](https://user-images.githubusercontent.com/53094530/218068677-6bd57989-6923-468a-a1f8-2470f12f5547.png)

2.- The vulnerability in english:

![image](https://user-images.githubusercontent.com/53094530/218068745-6b051e11-ea9f-46fb-a592-e2311e805fb4.png)

_AFTER MERGING the two vulnerabilities..._
![image](https://user-images.githubusercontent.com/53094530/218068900-430d6c5b-5fc2-4406-8ecf-67e03324d018.png)

1.- The vulnerability in spanish still has the references:
![image](https://user-images.githubusercontent.com/53094530/218069018-905fc6b3-878d-47ad-a10d-d7a1450a7cfd.png)

**2.- but the vulnerability in english has no longer references (!):**

![image](https://user-images.githubusercontent.com/53094530/218069100-f191ab50-66cb-4231-bba3-f7c32f115826.png)
